addons/hide-flyout/style.css

```
@@ -85,8 +85,8 @@
 }
 
 .sa-lock-object.locked .sa-lock-button {
-  background-color: var(--editorDarkMode-primary, #4d97ff);
-  border-color: var(--editorDarkMode-primary-variant, #3373cc);
+  background-color: var(--editorDarkMode-primary, #855cd6);
+  border-color: var(--editorDarkMode-primary-variant, #5846ab);
```

```suggestion
  border-color: var(--editorDarkMode-primary-variant, #714eb6);
```
This is the color used by vanilla Scratch: https://github.com/scratchfoundation/scratch-gui/blob/2d859b5/src/css/colors.css#L23

It will also be the value of the `primary-variant` CSS variable with default settings after #6074 is merged.